### DISCUSSION TITLE
Upgrade or downgrade plan.

### DISCUSSION THREAD

**Discussion by osvald0** [hyochan/react-native-iap#1610] [General]
Hello! Are there a way to upgrade or downgrade between subscriptions?
I was looking for information and it looks like there are not an option to handle this. 
What is the recommended approach?

Thanks!

**Comment by nl-danish** [comment]
Hey man, @osvald0 

Hope you are doing well.
Were you able to sucessfully implement upgrade and downgrade using this package? I know, it's been long so i just wanted to know.

Thanks.

**Comment by KartavyaParekh** [comment]
is there any update how to upgrade or downgrade subscription plan
